.gitignore

```diff
@@ -1,2 +1,4 @@
 target
 work
+.idea
```

Comment:
I don't think idea editor specific files needs to be added.

Reply:
I think you added it originally in https://github.com/jenkinsci/file-leak-detector-plugin/pull/3/commits/504dc08556e22aff4ea8083aecd422bbde2eab67 😄

I will go ahead and remove it, but for what it's worth `.idea` and other IntelliJ/Eclipse files are added to `.gitignore` by default for any plugins using the maven archetype generator: https://github.com/jenkinsci/archetypes/blob/master/empty-plugin/src/main/resources/archetype-resources/.gitignore. We could follow up with a separate PR to add all of those entries to `.gitignore` if desired.